src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -1398,18 +1364,34 @@ bool SlaterDetBuilder::readDetListH5(xmlNodePtr cur,
   CIcoeff.resize(ndets);
   ConfigTag.resize(ndets);
 
+#if defined(QMC_COMPLEX)
```

Comment:
In anbenali#12 I merge refactor this and the following opt coeffs read while eliminating the ifdefs.

Reply:
merged!